packages/blockchain-wallet-v4-frontend/src/modals/Xlm/SendXlm/FirstStep/ModalIcon/index.js

```diff
@@ -23,7 +23,7 @@ const ModalIconContainer = ({ className, showModal }) => (
 )
 
 const mapDispatchToProps = (dispatch, props) => ({
-  showModal: dispatch.bind(null, actions.modals.showModal(props.modal, { ...props }))
+  showModal: dispatch.bind(null, actions.modals.showModal({ props, type: props.modal }))
```

Comment:
`showModal` takes 3 arguments, but this seems to be passing it all as one object?

Reply:
yeah, this is more issue having non TS code in code base, good point I missed those :(